doc.go

```diff
@@ -118,6 +118,8 @@ The following connection parameters are supported:
   - tracing: Specifies the logging level to be used. Set to error by default.
     Valid values are trace, debug, info, print, warning, error, fatal, panic.
 
+  - disableQueryContextCache: disables parsing of query context returned from server and resending it to server as well.
```

Comment:
let's describe what's the default value for now

Reply:
Good idea, described.